osu.Game/Overlays/OverlayRulesetTabItem.cs

```diff
@@ -67,7 +71,7 @@ protected override void LoadComplete()
             Enabled.BindValueChanged(_ => updateState(), true);
         }
 
-        public override bool PropagatePositionalInputSubTree => Enabled.Value && !Active.Value && base.PropagatePositionalInputSubTree;
+        public override bool PropagatePositionalInputSubTree => Enabled.Value && base.PropagatePositionalInputSubTree;
```

Comment:
Note that this change also causes click sounds to play even when clicking the selected tab, but it's occurring on every other tab control so it should be fine.